cfg_mgmt/btp_application_certificate.py

```diff
@@ -208,7 +214,10 @@ def _extract_client_certificate(cert_response: dict) -> str:
     certs = pkcs7.load_pem_pkcs7_certificates(pkcs7_pem.encode('utf-8'))
     if not certs:
         raise ValueError('no certificates found in response')
-    return certs[0].public_bytes(serialization.Encoding.PEM).decode('utf-8')
+    allCerts = ""
```

Comment:
nit: [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names) suggests snake_case for variable names

Reply:
I would prefer to avoid mutation, e.g. with `.join()`:

```python
all_certs = ''.join([cert.public_bytes(...) for cert in certs])
```